Makefile

```diff
@@ -105,22 +120,26 @@ CFLAGS += -G 0 -non_shared -Xfullwarn -Xcpluscomm $(IINC) -nostdinc -Wab,-r4300_
 # Use relocations and abi fpr names in the dump
 OBJDUMP_FLAGS := -d -r -z -Mreg-names=32
```

Comment:
I think, as Dragorn pointed out in OoT, if we're making this optional we might as well use the long forms of the flags (leave -M, though, it's not any clearer as its long version).